x/bounty/client/cli/tx.go

```diff
@@ -359,3 +358,107 @@ func HostProcessFinding(cmd *cobra.Command, args []string) (fid uint64,
 
 	return fid, commentAny, hostAddr, nil
 }
+
+func NewReleaseFindingCmd() *cobra.Command {
```

Comment:
This was deprecated in favor of providing the decrypted plaintext for each finding to be verified by encrypting it again and matching it with the on-chain encrypted ciphertext

Reply:
so you'd only be able to release findings one by one

Reply:
make sense